src/Executor/Executor.php

```diff
@@ -26,9 +28,14 @@ class Executor
         'content-type' => '',
     ];
 
-    public function __construct(string $endpoint = 'http://appwrite-executor/v1')
-    { 
-        $this->endpoint = $endpoint;
+    public function __construct(string $endpoint = self::DEFAULT_HOST)
```

Comment:
No need for a default value, let's make maintainers more aware of this new value.